packages/shared/components/atoms/tiles/WalletTile.svelte

```diff
@@ -0,0 +1,47 @@
+<script lang="typescript">
+    import { Icon, Text } from 'shared/components'
+    import { FontWeightText } from 'shared/components/Text.svelte'
+    import { StardustAccount } from '@lib/typings/account'
+
+    export let wallet: StardustAccount
+
+    const alias = wallet?.meta?.alias ?? ''
+    const unclaimedBalance = wallet?.meta?.index === 0 || wallet?.meta?.index === 2 ? '12,345.678 SMR' : ''
```

Comment:
This is just some hard-coded data that will be removed once the actual app logic is hooked up.